openQA.spec

```diff
@@ -451,6 +453,7 @@ fi
 %{_datadir}/openqa/script/load_templates
 %dir %{_datadir}/openqa/lib
 %{_datadir}/openqa/lib/OpenQA/Client.pm
+%{_datadir}/openqa/lib/OpenQA/Client/*
```

Comment:
Did you test if this builds? I *think* you need to remove the /* at the end - but please double check

Reply:
> Not tested yet